.github/workflows/ci.yml

```diff
@@ -203,6 +203,7 @@ jobs:
             aiidalab_default_apps: >-
                 aiidalab-widgets-base@git+https://github.com/aiidalab/aiidalab-widgets-base@stable
                 quantum-espresso@git+https://github.com/aiidalab/aiidalab-qe@stable
+                sssp@git+https://github.com/aiidalab/aiidalab-sssp@main
```

Comment:
Is the main branch guaranteed to point to a stable release?

Reply:
The fact is the app doesn't have a stable release yet and for the near months `main` will keep being the stable branch with most new features.